ViewGenerator/build/ViewGenerator.targets

```diff
@@ -86,14 +85,12 @@
     <Exec Command="mklink /J &quot;$(ViewGeneratorToolsPath)\webpack\node_modules&quot; &quot;$(ViewPackerToolNodeModules)&quot;" />
   </Target>
   
-  <Target Name="VG_CalculateStylesheetInputs">
-    <ItemGroup>
-	  <VG_StyleSheetInputs Include="@(DefaultStyleSheet)"/>	
-	  <VG_StyleSheetInputs Condition="'@(DefaultStyleSheet)' != ''" Include="**\*.scss" Exclude="node_modules\**\*.scss" />
-    </ItemGroup>
-  </Target>
+<ItemGroup>
```

Comment:
Hum... this ItemGroup is not in a target? That does not create any problems? E.g. these properties always being empty